src/static/riot/analytics/analytics.tag

```diff
@@ -0,0 +1,540 @@
+<analytics>
+    <h1>Analytics</h1>
+
+
+    <h3>Date Range</h3>
+    <div class="ui blue buttons" ref="shortcut_buttons">
```

Comment:
I think dropdowns may be a better solution here. Then it is clear which date range and granularity is selected, and 
we can have both dropdowns on the same line and save some vertical space.

Reply:
Both button sets have been changed to dropdowns.